lib/index.js

```diff
@@ -161,14 +161,62 @@ App.prototype.start = function(cb) {
   }
 
   function connectPeers(done, results) {
+    // Get Max Rep from Networking.js
+    self.maxrep = self.maxrep || 3;
+
     results.loadPeers.forEach(function(peer) {
-      self.network.connect(peer.ip, peer.port, function() {
-        log.info('networking','Connecting to '+ peer.ip+':'+ peer.port);
+      self.network.connect(peer.value.ip, peer.value.port, function(err) {
+        // Evaluate A Connecting Peer
+        if (!err)
+          evalGoodPeer(peer);
+        // Evaluate A Non-Connecting Peer
+        else
+          evalBadPeer(peer);
       });
     });
     done();
   }
 
+  function evalGoodPeer(peer) {
```

Comment:
see if you can `evalGoodPeer` and `evalBadPeer` into one function `evalPeer(rating, peer)`  . you  only need to write `self.peersDB.put` one.